command/staticcommands/shop.py

```diff
@@ -17,13 +17,23 @@ def execute(self):
                 buytem = buytem.lower()
                 if buytem == ('potion'):
                     potion = items.consumable('Potion', 1, 10, 10)
-                    if self.assetContainer.player.gold < potion.cost:
-                        print "You don't have enough gold to buy this!"
+                    quantity = raw_input("How many would you like to buy?: ")
+                    current = 0
+                    if quantity.isalnum():
```

Comment:
`isalnum()` returns true if alphanumeric, you might want to consider `isdigit()` which would only be true for numbers